src/main/java/ToDo.java

```diff
@@ -3,8 +3,17 @@ public ToDo(String description) {
         super(description);
     }
 
+    protected static ToDo stringToToDo(String s) {
+        return new ToDo(s.substring(12));
+    }
```

Comment:
Try to add in JavaDoc in all of your methods. In particular, you probably will need JavaDoc for this method as it is difficult to understand what it is doing from the naming (to-to-do).